client/src/containers/NavBarContianer.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+import NavBar from '../components/Header/NavBar/NavBar';
+import userStore from '../stores/userStore';
+
+const NavBarContainer = (): JSX.Element => {
+  const navItems = [
+    {
+      text: userStore.user ? '로그아웃' : '로그인',
+      path: userStore.user ? '/logout' : '/login',
```

Comment:
```suggestion
...(isNotNone(userStore.user) ? { text: '로그아웃', path: '/logout' } : { text: '로그인', path: '/login' }
```

이렇게 text, path를 객체로 묶어서 `spread`로 뿌려주는건 어떤가요?
(위 코드를 조금 다듬어서?)

Reply:
아 수정이 필요한 부분인 것 같네요. 감사합니다.

아예 함수를 따로 빼서 가져오게끔 수정해보았습니다. :)